packages/web/src/common/store/cache/collections/utils/processAndCacheCollections.ts

```diff
@@ -40,7 +40,7 @@ export function* processAndCacheCollections(
   yield addTracksFromCollections(collections)
 
   let reformattedCollections = collections.map((c) =>
-    reformat(c, audiusBackendInstance)
+    reformatCollection({ collection: c, audiusBackendInstance })
```

Comment:
Oh wait, I see. The cache already does this, so by calling `addEntries` in audius-query, we already do it.

Have you tried it without calling this in the endpoint's fetch implementation? It probably helps the types match up, but I think the data should still be there if you don't. We should be saving the results to cache which internally calls `reformatCollection` then selecting the cache data as the return from the hook.